### DISCUSSION TITLE
Crazy with esp-32 wroom

### DISCUSSION THREAD

**Discussion by jgmonm** [BitMaker-hub/NerdMiner_v2#394] [Q&A]
Good afternoon, everyone. I have a problem with the wifi of the sp-32 wroom board. Let me tell you. I've programmed it with the nerdminer software that's here on the web. The first time it went well, but I put the data that corresponded badly and when I started the esp-32 wroom, it asks me ssid and wifi password to access the configuration. I've deleted it, I've put the factory binary on it, but every time I try to connect with the phone's wifi to set it up, it asks me for ssid and password and I don't know what else to do. It always asks me for a password to access the settings. Without further ado, and thanking you for your help, a greeting.

**Comment by canadamadman** [comment]
the wifi ssid and the password are listed on the main page.
